core/kazoo_config/test/kapps_config_migrate_test.erl

```diff
@@ -0,0 +1,390 @@
+-module(kapps_config_migrate_test).
+
+-include_lib("eunit/include/eunit.hrl").
+
+-define(WHAPPS_CONTROLLER
+       ,<<"{
```

Comment:
why not storing it as a fixture file and load it?

Reply:
Is that on 4.1?

Reply:
I didn't mean FixtureDB, just save it as a JSON file in `test` directory and read/decode it in your test function.